contrib/devtools/symbol-check.py

```diff
@@ -278,6 +279,38 @@ def check_PE_subsystem_version(binary) -> bool:
         return True
     return False
 
+MANIFEST_TEMPLATE = ('^' + re.escape("""<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
```

Comment:
Why do we need this check? Regex over a copy-pasted blob of XML is pretty meh. We aren't really checking anything important here either?

Reply:
Currently, no, but after #32380 it becomes critical because it determines the process code page.

> (not too important yet now as we're just adding some metadata, but when we add critical things in there like "use UTF-8 codepage" as in https://github.com/bitcoin/bitcoin/pull/32380, it's really important for them to be in the release too)

This is a quick and dirty check that the manifest is added and is what we expect, feel free to add detailed XML parsing.

Reply:
It also already found one bug in this PR: the manifest was not added to `test_bitcoin.exe` .

Reply:
I think for now we could just assert `binary.resources_manager.has_manifest`, if we care that this is being added? Could save anything else for if/when it's actually being used?

Reply:
i don't see the problem with having it right now like this, but okay.

Reply:
One thing i did think about is using the input file (`make/windows-app.manifest.in`) as template instead of hardcoding it here, but having the check depend on files scattered throughout the repository is probably not that great, either.

Reply:
> I think for now we could just assert `binary.resources_manager.has_manifest`, if we care that this is being added? Could save anything else for if/when it's actually being used?

Thanks! [Reworked](https://github.com/bitcoin/bitcoin/pull/32396#issuecomment-2862815509).